src/components/Modals/CreateItemModal/CreateItemModal.types.ts

```diff
@@ -42,6 +42,16 @@ export type CreateItemModalMetadata = {
   changeItemFile?: boolean
 }
 
+export type ItemAssetJson = {
+  name?: string
+  description?: string
+  category?: WearableCategory
+  rarity?: ItemRarity
+  thumbnail?: string
+  model?: string
+  bodyShape?: BodyShapeType
```

Comment:
If you create an `asset.json` that has some things like `bodyShape` as "Female", that is, the value it should have but uppercased, the value is set to "Female" (which is wrong) and the UI seems to let you create the item with it.
It think that we should have a minimal way to check the values input here because it might cause some issues.
![Screen Shot 2021-11-15 at 16 03 42](https://user-images.githubusercontent.com/1120791/141839302-fcf0ad31-49cb-4032-80cf-1e7c63a78e86.png)

Reply:
Following the same idea, if the model path is wrong, the process looks for any of the GLBs but IMO it should throw to let you know that you made a mistake.

Reply:
good idea! I added validation for file paths and enum values:
<img width="688" alt="Screen Shot 2021-11-16 at 4 34 39 PM" src="https://user-images.githubusercontent.com/2781777/142053546-047832f8-ca7c-4bda-92ea-ca2cf1406b71.png">
<img width="687" alt="Screen Shot 2021-11-16 at 4 54 46 PM" src="https://user-images.githubusercontent.com/2781777/142056290-a1f4a719-152d-435b-a9dc-c697d2774f38.png">